_posts/2025-04-06-mjh5810-nixos-frp.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+
+categories:
+- Contribution
+
+author: Max Horton
+
+title: "Improving Documentation in NixOS Modules"
+
+----
```

Comment:
you have an extra dash here thats breaking stuff

Reply:
![Screenshot_20250406_212254](https://github.com/user-attachments/assets/4cb4d89c-f60a-4d83-a55d-079c508930c5)